### PR TITLE
ui-speedspacechart: helpers fix spelling in comment

### PR DESCRIPTION
**Pull request by SergeCroise**
- progress**ive**ly
see:
https://www.merriam-webster.com/dictionary/progressively
https://www.merriam-webster.com/thesaurus/progressively

### CODE REVIEW

**Comment by SergeCroise** [comment]
@flomonster , @emersion please review !

File: ui-speedspacechart/src/components/helpers/drawElements/tickX.ts [hunk 1]
```diff
@@ -59,7 +59,7 @@ export const drawTickX = ({ ctx, width, height, store }: DrawFunctionParams) =>
       const textPositionX = MARGIN_LEFT + xPosition * i + CURVE_MARGIN_SIDES / 2;
       let opacity = 1;
 
-      // low progressivily opacity for text when text is near borders or cursor, except for 0
+      // low progressively opacity for text when text is near borders or cursor, except for 0
```
Comment (emersion) [inline ui-speedspacechart/src/components/helpers/drawElements/tickX.ts:62]: The spelling is now correct but the grammar still doesn't seem to be… Maybe the intent of the comment was "reduce progressively opacity [...]"?

Comment (SergeCroise) [inline ui-speedspacechart/src/components/helpers/drawElements/tickX.ts:62]:
```suggestion
      // reduce progressively opacity for text when text is near borders or cursor, except for 0
```

**Comment by flomonster** [comment]
I have a complete code refactoring of this file [here](https://github.com/OpenRailAssociation/osrd-ui/pull/436). I'll apply it to my PR directly.

**Pull request closed without merging**